contracts/UsulVetoGuard.sol

```diff
@@ -0,0 +1,147 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./interfaces/IUsulVetoGuard.sol";
+import "./interfaces/IVetoVoting.sol";
+import "./interfaces/IOZLinearVoting.sol";
+import "./interfaces/IUsul.sol";
+import "./TransactionHasher.sol";
+import "./FractalBaseGuard.sol";
+import "@gnosis.pm/zodiac/contracts/factory/FactoryFriendly.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+/// @notice A guard contract that prevents transactions that have been vetoed from being executed a Gnosis Safe
+/// @notice through a Usul module with an attached OZLinearVoting strategy
+contract UsulVetoGuard is
```

Comment:
Ideally this can be made generic, but otherwise we should change the name to reflect the voting strategy it guards, since every later strategy will also be a UsulVetoGuard

Reply:
Fixed this, the UsulVetoGuard is now compatible with any Usul voting strategy